x-pack/plugins/infra/public/common/visualizations/lens/dashboards/asset_details/host/host_metric_charts.ts

```diff
@@ -11,207 +11,383 @@ import { hostLensFormulas } from '../../../../constants';
+          label: i18n.translate('xpack.infra.assetDetails.metricsCharts.metric.label.write', {
+            defaultMessage: 'Write',
+          }),
         },
-        type: 'visualization',
+      ],
+      options: {
+        seriesType: seriesType.Area,
       },
-    ],
-    overrides: {
-      settings: XY_OVERRIDES.settings,
+      type: LayerParamsType.Visualization,
     },
-    dataViewOrigin: 'metrics',
+  ],
+  overrides: {
+    settings: XY_OVERRIDES.settings,
   },
-  {
-    id: 'rxTx',
-    title: i18n.translate('xpack.infra.assetDetails.metricsCharts.network', {
-      defaultMessage: 'Network',
-    }),
-    layers: [
-      {
-        data: [
-          {
-            ...hostLensFormulas.rx,
-            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.network.label.rx', {
-              defaultMessage: 'Inbound (RX)',
-            }),
-          },
-          {
-            ...hostLensFormulas.tx,
-            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.network.label.tx', {
-              defaultMessage: 'Outbound (TX)',
-            }),
-          },
-        ],
-        options: {
-          seriesType: 'area',
+  dataViewOrigin: DataViewOrigin.Metrics,
+};
+
+const rxTx = {
+  id: 'rxTx',
+  title: i18n.translate('xpack.infra.assetDetails.metricsCharts.network', {
+    defaultMessage: 'Network',
+  }),
+  layers: [
+    {
+      data: [
+        {
+          ...hostLensFormulas.rx,
+          label: i18n.translate('xpack.infra.assetDetails.metricsCharts.network.label.rx', {
+            defaultMessage: 'Inbound (RX)',
+          }),
         },
-        type: 'visualization',
+        {
+          ...hostLensFormulas.tx,
+          label: i18n.translate('xpack.infra.assetDetails.metricsCharts.network.label.tx', {
+            defaultMessage: 'Outbound (TX)',
+          }),
+        },
+      ],
+      options: {
+        seriesType: seriesType.Area,
       },
-    ],
-    overrides: {
-      settings: XY_OVERRIDES.settings,
+      type: LayerParamsType.Visualization,
     },
-    dataViewOrigin: 'metrics',
+  ],
+  overrides: {
+    settings: XY_OVERRIDES.settings,
   },
+  dataViewOrigin: DataViewOrigin.Metrics,
+};
+
+export const hostMetricCharts: Array<
+  Pick<TypedLensByValueInput, 'id' | 'title' | 'overrides'> & {
+    dataViewOrigin: DataViewOrigin;
+    layers: XYChartLayerParams[];
+  }
+> = [
+  cpuUsage,
+  memoryUsage,
+  normalizedLoad1m,
+  logRate,
+  diskSpaceUsageAvailable,
+  diskThroughputReadWrite,
+  diskIOReadWrite,
+  rxTx,
+];
+
+export const hostMetricChartsFullPage: Array<
+  Pick<TypedLensByValueInput, 'id' | 'title' | 'overrides'> & {
+    dataViewOrigin: DataViewOrigin;
+    layers: XYChartLayerParams[];
+  }
+> = [
+  cpuUsage,
+  cpuUsageBreakdown,
+  memoryUsage,
+  memoryUsageBreakdown,
+  normalizedLoad1m,
+  loadBreakdown,
+  logRate,
+  diskSpaceUsageAvailable,
+  diskSpaceUsageByMountPoint,
+  diskThroughputReadWrite,
+  diskIOReadWrite,
+  rxTx,
```

Comment:
Sweet. We could extract the chart configuration above these lines in a separate file. But this is nice already

Reply:
Moved the charts configuration [to a separate folder](https://github.com/elastic/kibana/pull/164538/files#diff-db752591a856a6bf868db420dc16ee2217ac6a0305120de93631a3e1cac4ae48) and files as discussed